sleap/gui/commands.py

```diff
@@ -2026,6 +2026,34 @@ def do_action(context: CommandContext, params: dict):
         # is not visually apparent which session is selected after clicking the button
         context.state["selected_session"] = None
 
+        # Find parent of calibration file
+        calibration_path = Path(camera_calibration)
+        parent_dir = calibration_path.parent
+        # Use camcorder names in session to find camera folders
+        cameras = session.camera_cluster.cameras
+        camera_names = [camera.name for camera in cameras]
+
+        # Find videos inside camera folders
+        video_paths = []
+        for camera_name in camera_names:
+            camera_folder = parent_dir / camera_name
+            print(camera_folder)
+
+            # Skip if camera folder does not exist
+            if not camera_folder.exists():
+                continue
+
+            # Append all videos in camera folder
+            video_path = None
+            for file in camera_folder.iterdir():
+                if file.endswith(".mp4"):
```

Comment:
We also want to check for common supported video file types, not just mp4 (although that is a good start!)